core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -476,15 +468,9 @@ public void throwFileNotFoundExceptionWhenListStatusNonExistingTest() throws Exc
     try {
       Path path = new Path("/ALLUXIO-2036");
       alluxio.client.file.FileSystem alluxioFs = mock(alluxio.client.file.FileSystem.class);
-      alluxioHadoopFs = new FileSystem(alluxioFs);
-      URI uri = URI.create(Constants.HEADER + "host:1");
-      alluxioHadoopFs.initialize(uri, getConf());
-      ListStatusPOptions listStatusPOptions = ListStatusPOptions.getDefaultInstance().toBuilder()
-          .setExcludeMountInfo(alluxioHadoopFs.mAlluxioConf.getBoolean(
-              PropertyKey.USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS)).build();
-      when(alluxioFs.listStatus(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path)),
-          listStatusPOptions))
+      when(alluxioFs.listStatus(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))))
           .thenThrow(new FileNotFoundException("ALLUXIO-2036 not Found"));
+      alluxioHadoopFs = new FileSystem(alluxioFs);
       FileStatus[] fileStatuses = alluxioHadoopFs.listStatus(path);
```

Comment:
Looks like the `fileStatuses` here is unused. Could you remove it?

Reply:
This operation is required to throw an exception for test.